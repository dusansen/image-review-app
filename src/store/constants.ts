export const LOAD_IMAGE = 'LOAD_IMAGE';
export const IMAGE_LOADED = 'IMAGE_LOADED';
export const IMAGE_LOADING_FAILED = 'IMAGE_LOADING_FAILED';
