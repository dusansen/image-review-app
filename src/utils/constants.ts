export const LIKED_IMAGES_KEY = 'likedImages';
export const DECLINED_IMAGES_KEY = 'declinedImages';
