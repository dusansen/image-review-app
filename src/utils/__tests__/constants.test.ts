import * as constants from '../constants';

describe('Utils constants', () => {
  it('should match snapshot', () => {
    expect(constants).toMatchSnapshot();
  });
});
