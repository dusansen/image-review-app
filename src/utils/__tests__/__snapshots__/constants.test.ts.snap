// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils constants should match snapshot 1`] = `
Object {
  "DECLINED_IMAGES_KEY": "declinedImages",
  "LIKED_IMAGES_KEY": "likedImages",
}
`;
